resources/views/statuslabels/edit.blade.php

```diff
@@ -71,11 +71,14 @@
 
 @section('moar_scripts')
     <!-- bootstrap color picker -->
+    <?php 
+    $color = request()->old('color', $item->color) ? request()->old('color', $item->color) : '#AA3399';
+    ?>
     <script nonce="{{ csrf_token() }}">
 
         $(function() {
             $('.color').colorpicker({
-                color: '#AA3399',
+                color: '{{ $color }}',
```

Comment:
Something like 

```php
color: '{{ old('color', $item->color) ? request()->old('color', $item->color) : '#AA3399' }}',
```

Reply:
I completely agree with you. We should indeed follow best practices when working with Laravel and its blade templating engine. Here's how we could modify the function:

`color: `{{ old('color', $item->color) ?: '#AA3399' }}`,`

As ` request()->old() ` is not necessary if we are already using the old() helper.